moveit_kinematics/kdl_kinematics_plugin/CMakeLists.txt

```diff
@@ -1,9 +1,16 @@
 set(MOVEIT_LIB_NAME moveit_kdl_kinematics_plugin)
 
+find_package(orocos_kdl REQUIRED)
```

Comment:
orocos_kdl is already required by the top-level CMakeLists.txt [here](https://github.com/ros-planning/moveit2/blob/main/moveit_kinematics/CMakeLists.txt#L10). I also think that the other changes in this file are redundant, `ament_target_dependencies()` already handles include directories and linked libraries.